docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -89,13 +91,207 @@ For the complete list of actions that {ilm} can perform, see <<ilm-actions>>.
 
 [discrete]
 [[ilm-gs-apply-policy]]
-=== Create an index template to apply the lifecycle policy
+=== Create a composable template to create the data stream and apply the lifecycle policy
+
+To create a data stream we'll create a composable template with a `data_stream` definition
+that will specify the lifecycle policy.
```

Comment:
Nit: Avoid "we" and future tense where possible.

```suggestion
To set up a data stream, first create a composable template to specify the lifecycle policy. Because
the template is for a data stream, it must also include a `data_stream` definition.
```

I wonder if we should mention other requirements here, specifically a timestamp mapping.

Reply:
Starting the sentence with "Because" sounds awkward to me :D my suggestion:
```suggestion
To set up a data stream, first create a composable template to specify the lifecycle policy. A composable template for used a data stream must include a `data_stream` definition.
```

Reply:
I'm good with Lee's suggestion. Will leave mine up for context.

Reply:
```A composable template for used a data stream must include a `data_stream` definition.```
This doesn't make sense to me. Is it meant to be ?
```
For a composable template to be used with a data stream it must include a `data_stream` definition
```